src/os/ObjectStore.h

```diff
@@ -761,9 +761,15 @@ class ObjectStore {
    *
    * @return iterator, null on error
    */
+  struct omap_iter_seek_t {
+    std::string key;
+    bool lower_bound = true;
```

Comment:
seek_type, enum with LOWER_BOUND and UPPER_BOUND

Reply:
Switched.